--- docker-entrypoint.sh	2018-01-13 23:19:01.157936404 -0500
+++ entrypoint_modif	2018-01-13 23:18:06.406219821 -0500
@@ -160,7 +160,7 @@
 			echo "CREATE USER '$MYSQL_USER'@'%' IDENTIFIED BY '$MYSQL_PASSWORD' ;" | "${mysql[@]}"
 
 			if [ "$MYSQL_DATABASE" ]; then
-				echo "GRANT ALL ON \`$MYSQL_DATABASE\`.* TO '$MYSQL_USER'@'%' ;" | "${mysql[@]}"
+				echo "GRANT ALL ON *.* TO '$MYSQL_USER'@'%' WITH GRANT OPTION ;" | "${mysql[@]}"
 			fi
 
 			echo 'FLUSH PRIVILEGES ;' | "${mysql[@]}"
